src/renault_api/cli/charge/control.py

```diff
@@ -47,5 +47,37 @@ async def start(
     vehicle = await renault_vehicle.get_vehicle(
         websession=websession, ctx_data=ctx_data
     )
-    response = await vehicle.set_charge_start()
+    await vehicle.get_details()
+    if (
+        vehicle._vehicle_details
+        and vehicle._vehicle_details.get_model_code() == "XBG1VE"
+    ):
+        response = await vehicle.set_charge_pause_resume("resume")
+    else:
+        response = await vehicle.set_charge_start()
     click.echo(response.raw_data)
+
+
+@click.command()
+@click.pass_obj
+@helpers.coro_with_websession
+async def stop(
+    ctx_data: Dict[str, Any],
+    *,
+    websession: aiohttp.ClientSession,
+) -> None:
+    """Stop charge."""
+    vehicle = await renault_vehicle.get_vehicle(
+        websession=websession, ctx_data=ctx_data
+    )
+    await vehicle.get_details()
+    if (
+        vehicle._vehicle_details
+        and vehicle._vehicle_details.get_model_code() == "XBG1VE"
+    ):
```

Comment:
```suggestion
    adapter = await vehicle.get_car_adapter()
    if adapter.carGateway  == "XXX":
```

Reply:
Since the carGateway cannot be uses (it shows AVN also for the dacia spring), I suggest this:
```suggestion
    details = await vehicle.get_details()
    if details.get_model_code() == "XBG1VE":
```